### PR TITLE
Added github actions to replace travis

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2123

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+       matrix:
+         os: [ubuntu-20.04]
+         python-version: ['3.5','3.8']
+         toxenv: ['django22', 'quality']
+         exclude:
+           - python-version: '3.5'
+             toxenv: 'quality'
+
+     steps:
+       - uses: actions/checkout@v2
```
Comment (mraarif) [inline .github/workflows/ci.yml:25]: you might want to switch to using `actions/checkout@v1`, we've faced issues with coverage checks in some other repos with checkout v2

**Review by mraarif — APPROVED** [review]
a small suggestion

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Jawayria) [reply .github/workflows/ci.yml:25]: Should I make this change in all repos I have added GitHub Actions to? Or its specific to this one?

Comment (mraarif) [reply .github/workflows/ci.yml:25]: in all the repos where we've coverage checks at least

Comment (Jawayria) [reply .github/workflows/ci.yml:25]: Okay got it

**Review by jmbowman — APPROVED** [review]

**Pull request merged**